memory/ndctl.py

```diff
@@ -358,9 +358,9 @@ def test_multiple_ns_multiple_region(self):
             self.log.info("Using %s for muliple namespaces", region)
             self.log.info("Creating %s namespaces", self.cnt)
             if not self.size:
-                size = self.get_json_val(self.get_json(
+                self.size = self.get_json_val(self.get_json(
```

Comment:
one question why we need to change scope of variable as seems me it used only one place or it is bug fix ?

Reply:
yes, it is a fix.

Reply:
ok sure